presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -184,6 +184,7 @@ SystemConfig::SystemConfig() {
           NUM_PROP(kHeartbeatFrequencyMs, 0),
           STR_PROP(kExchangeMaxErrorDuration, "30s"),
           STR_PROP(kExchangeRequestTimeout, "10s"),
+          STR_PROP(kExchangeConnectTimeout, "0S"),
```

Comment:
What if connector timeout is set to zero? Infinite wait for connection setup? Thanks!

Reply:
Yeah